.github/workflows/reusable-deploy.yml

```
@@ -0,0 +1,181 @@
+  workflow_call:
+    inputs:
+      safe-address:
+        description: 'Gnosis Safe multisig address'
+        required: true
+        type: string
+
+      network:
+        description: 'Network to deploy to'
+        required: true
+        type: string
+        default: 'sepolia'
```

actionlint (1.7.8)

input "network" of workflow_call event has the default value "sepolia", but it is also required. if an input is marked as required, its default value will never be used

(events)